test/Controller/ArticleFiguresControllerTest.php

```diff
@@ -47,12 +47,13 @@ public function it_displays_an_article_figures_page_for_a_vor()
         $tables = $figureTypes->eq(2)->filter('.asset-viewer-inline');
         $this->assertSame('Table 1 label', trim($tables->eq(0)->filter('.asset-viewer-inline__header_text')->text()));
 
-        $this->assertSame('Data sets', $figureTypes->eq(3)->filter('.article-section__header_text')->text());
-        $dataSets = $figureTypes->eq(3)->filter('.article-section__body')->children();
-        $this->assertSame('The following data sets were generated', trim($dataSets->eq(0)->text()));
-        $this->assertSame('Data set 1', trim($dataSets->eq(1)->filter('.reference__title')->text()));
-        $this->assertSame('The following previously published data sets were used', trim($dataSets->eq(2)->text()));
-        $this->assertSame('Data set 2', trim($dataSets->eq(3)->filter('.reference__title')->text()));
+        $this->assertSame('Data availability', $figureTypes->eq(3)->filter('.article-section__header_text')->text());
```

Comment:
the progressive numbering is a bit fragile but given the different `filter()` calls I don't see a way to unify those